.gitignore

```diff
@@ -8,6 +8,9 @@
 # testing
 /coverage
 
+# quiz data
+/src/plays/quizeo/src/data
```

Comment:
Why data is added in gitignore?

Reply:
The file contains all the questions and its answers data.

Reply:
Don't we need that in production?